project-addons/custom_partner/partner_view.xml

```diff
@@ -250,6 +249,29 @@
                 <xpath expr="//field[@name='unreconciled_aml_ids']/tree/field[@name='move_id']" position="after">
                     <field name="stored_invoice_id"/>
                 </xpath>
+                <xpath expr="//field[@name='unreconciled_aml_ids']/tree/field[@name='result']" position="after">
+                    <field name="amount_currency"/>
+                    <field name="currency_id"/>
+                </xpath>
```

Comment:
Esto se podría eliminar ya que nunca se van a ver rellenado en el listado de seguimiento de pagos, ya que nunca tenéis cobros de clientes en una divisa distinta a la por defecto